src/dlgAboutDialog.cpp

```diff
@@ -1008,7 +1008,7 @@ void dlgAboutDialog::setThirdPartyTab(const QString& htmlHead) const
 void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
 {
     // see https://www.patreon.com/mudlet if you'd like to be added!
-    QStringList mightier_than_swords = {"Joshua C. Burt", "Qwindor Rousseau", "Maiyannah Bishop"};
+    QStringList mightier_than_swords = {"Joshua C. Burt", "Maiyannah Bishop", "Qwindor Rousseau", "Stick In the MUD 🌯"};
```

Comment:
I think a comment explaining/documenting the use of a U+1F32F {BURRITO} grapheme would be useful here. Remember it will look different on different OSes and it might be considered **[Mojibake](https://en.wikipedia.org/wiki/Mojibake)** in some situations...

Reply:
Oh. I was hoping it's a microphone, not a burrito.

Reply:
Fixed. As to the question of why - that's why Git keeps history for us. Trace it back to this PR and you'll know!